docs/Developers/Writing_Native_Bears.rst

```diff
@@ -531,16 +531,16 @@ can be used as follows:
 Aspect Bear
 -----------
 
-Aspect is a feature in coala that make configuring coala in project more easy
-and language agnostic. For more detail about aspect, see cEP-0005 in
+Aspect is a feature in coala that make configuring coala in projects easier
+and language agnostic. For more details about aspects, see cEP-0005 in
 https://github.com/coala/cEPs/blob/master/cEP-0005.md.
 
-An aspect-compliant bear MUST:
+An aspect-compliant bear must:
 
-1. Declare list of aspect it can fix and detected. Note that the aspect MUST be
-   a leaf aspect. You can see list of supported aspect here
+1. Declare a list of aspect it can detect and fix. Note that the aspect must be
```

Comment:
I think it should be ```a list of aspects```. Please confirm with a maintainer.

Reply:
@jayvdb even i am confused with with it. is aspect fine or should it be aspects?

Reply:
plural is ok IMO